projects/plugins/jetpack/functions.opengraph.php

```diff
@@ -604,16 +621,13 @@ function jetpack_add_fediverse_creator_open_graph_tag( $tags ) {
 				continue;
 			}
 
-			// Did we skip this connection for this post?
-			if ( get_post_meta( $post->ID, $publicize->POST_SKIP_PUBLICIZE . $connection_id, true ) ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
-				continue;
-			}
+			$last_share = end( $shares_to_connection );
```

Comment:
In the meta there can be multiple shares to the same connection, we can consider the last one as that's the most recent